benchmarks/dynamo/common.py

```diff
@@ -1522,6 +1522,14 @@ def parse_args(args=None):
     parser.add_argument(
         "--ci", action="store_true", help="Flag to tell that its a CI run"
     )
+    parser.add_argument(
+        "--dynamic-ci-skips",
```

Comment:
nit: it makes the name worst but maybe a `-only` would make it clear that it only runs the dynamic shape skips. The doc below could be clearer as well.